stories/content.stories.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Content from './../src/web/components/Content';
+
+// TODO remove html entities when they are implemented
+const content = [
```

Comment:
I have several elements without root component when I put it between paranthesis, it wants a root component. And I didn't want an intermediate component between my typographic tags and the Content component.
That's why I chose to put them in the parathesis. I saw that there were HOCs that would allow you to do the same thing without array.
If it's a bad practice, I can change.